docs/4.3/aws_oss_guide.md

```diff
@@ -386,12 +379,308 @@ To upgrade to a newer version of Teleport:
 - Copy  and its contents should also be backed up and copied over to the new instance.
 - Either restart the instance, or log in via SSH and run `sudo systemctl restart teleport.service`
 
-# Running Teleport Enterprise on AWS
-Most of this guide has been designed for OSS Teleport. Most of this guide also applies to Teleport Enterprise
-with a few extra notes around adding a license and getting the correct binary. If you would
-like help setting up Teleport Enterprise on AWS, please mail us at info@gravitational.com
+## Accessing EKS using Teleport
```

Comment:
optional: it would be useful to provide a brief overview of what this section really does:
"In this section, we will set Kubernetes RBAC permissions needed by Teleport and configure the EC2 instance running Teleport to map to those permissions".

Reply:
Added